### DISCUSSION TITLE
Question regarding example in documentation

### DISCUSSION THREAD

**Discussion by saharshbhansali** [polybar/polybar#2686] [Q&A] [answered]
I was trying to set up programs to launch on clicking and came across an issue in the documentation

https://polybar.readthedocs.io/en/stable/user/actions.html#menu-example

The example, which I believe intended to show a before and after of the updated formats, just repeats the same thing again.
I apologize if I am posting this in the wrong section, as I couldn't find / figure out how to raise the issue properly.
I believe I can resolve this issue too, but am unsure how to raise it with the devs. Can someone please help me out?

**Comment by patrick96** [comment, marked as answer]
You're posting in the right place :)

But I'm not sure I'm seeing the issue.
The first module config uses (the deprecated) `menu-open-1` and `menu-open-2` and the second config uses the new format `#apps.open.1` and `#apps.open.2`

**Reply by saharshbhansali** [reply]
I did copy paste the custom module to check it out, but I still got errors. I must have made an error while copy pasting... I'll test it out and get back to you soon.
Thank you very much!

**Reply by patrick96** [reply]
That config snippet isn't really meant for copy-pasting. It's more of an illustration for how to migrate existing modules. In this case it seems the snippet is missing the `label-open` setting.

For completeness, we should probably add it to the documentation.

**Reply by saharshbhansali** [reply]
Yeah, that would be nice. Most newbies (like me, maybe?) would usually use the docs as a start off point and customize after that, cause that's probably the fastest way to get it done.
Should I try and add one and open a PR?

**Reply by patrick96** [reply]
I already fixed the example in readthedocs. Also, there is also an example on the [wiki](https://github.com/polybar/polybar/wiki/Module:-menu).

**Reply by saharshbhansali** [reply]
That's great, then! I'll mark the question as answered. 
Thanks again!
